packages/manager/apps/pci-kubernetes/src/pages/detail/log/components/LogTiles.component.tsx

```diff
@@ -41,7 +41,11 @@ export function LogTiles({ projectId, kubeId }: Readonly<LogTilesProps>) {
     projectId,
     kubeId,
   );
-  if (isLogsPending) return undefined;
+
+  if (isLogsPending) {
+    return <></>;
```

Comment:
why not return null instead of fragment ?

Reply:
A React fragment is a JSX element, not shown in the DOM, and it gets the job done.